modules/Robot.cpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file   Robot.cpp
+ * @brief  外部リソースのインスタンスを管理するクラス
+ * @author takuchi17
+ */
+
+#include "Robot.h"
+
+Robot::Robot() : motorController(), cameraCapture() /*, colorSensor()*/ {}
```

Comment:
直接公式のAPIのインスタンスを生成するときは、ポートを指定する必要があります。

```suggestion
Robot::Robot() : motorController(), cameraCapture() , colorSensor(EPort::PORT_E) {}
```